### PR TITLE
Irc full name overlap

### PR DESCRIPTION
**Pull request by UwUnyaa**
This is intended to fix https://github.com/vector-im/element-web/issues/20757

Notes: Add a background to expanded nick name in IRC layout to make it readable.

Tags: T-enhancement

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Add a background to expanded nick name in IRC layout to make it readable. ([\#7652](https://github.com/matrix-org/matrix-react-sdk/pull/7652)). Fixes vector-im/element-web#20757. Contributed by @UwUnyaa.

### CODE REVIEW

File: res/css/views/rooms/_IRCLayout.scss [hunk 1]
```diff
@@ -198,6 +198,10 @@ $irc-line-height: $font-18px;
 
         > .mx_SenderProfile_displayName {
```
Comment (SimonBrandner) [inline res/css/views/rooms/_IRCLayout.scss:199]: How would this work for disambiguated users (users that share the same display name, so we also show their `mxid`)?

**Review by SimonBrandner — COMMENTED** [review]
Could you please include a screenshot/video of how this looks in action to aid review?

**Comment by UwUnyaa** [comment]
Here's a screenshot of (now fixed) mxid:
image

Here's a video of how changed UI works:
https://user-images.githubusercontent.com/16294095/151383382-ca6cc75b-7caf-4585-9d36-a59c879141b4.mp4

**Review by SimonBrandner — APPROVED** [review]
Thanks!

**Review by andybalaam — APPROVED** [review]

**Pull request merged**